.travis.yml

```diff
@@ -62,24 +60,24 @@ before_script:
 
 script:
   - >
-    if [[ $TRAVIS_PYTHON_VERSION_MINOR < 4 || $TRAVIS_PYTHON_VERSION_MAJOR == 2 ]]; then
+    if [[ "$UNSUPPORTED" == true ]]; then
       coverage run setup.py install | grep -q 'coala supports only python 3.4 or later'
+    fi
+  - if [[ "$UNSUPPORTED" == true ]]; then exit; fi
+  - bash .misc/tests.sh
+  - python setup.py bdist_wheel
+  - pip install ./dist/coala-*.whl
+  - pip install coala-bears[alldeps] --pre -U
+  # https://github.com/coala/coala-bears/issues/1037
```

Comment:
This is an indentation only change, and should not be changed in this commit.
You can read more about it at https://github.com/coala/coala-bears/issues/1037